.github/actions/setup-environment/action.yml

```diff
@@ -92,6 +96,10 @@ runs:
         fi
       shell: bash
       working-directory: "./eden"
+    - name: Set eve log level
+      run: ./eden config set default --key=eve.log-level --value=${{ inputs.eve_log_level }}
```

Comment:
As I remember there were two [options](https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md) for tests: one to collect inside EVE-OS, another one to send to controller. Can you please validate, that your changes are enough?

Reply:
thank you for the suggestion @giggsoff !

the comments in the [template](https://github.com/lf-edge/eden/blob/e974c0afc9b817350a2d15301b41b4954ca54dd2/pkg/defaults/templates.go#L157-L162) say it's the right field to change, since I want to have this log level to be captured in the artifacts that Eden workflows produce (in collect_logs mostly).

Reply:
Thank you for validation!
Note, that If you say about the step [here](https://github.com/lf-edge/eden/blob/8545f10ca982fc9b621aab74616eb03f1a217e12/.github/actions/publish-logs/action.yml#L12), then they come from controller (Adam).

Reply:
Ah, I see. It should work for extraction logs directly from device's store.
The store has its rotation, but it should be enough for tests for sure.